### PR TITLE
Atomic: Update how we fetch the plan slug

### PR DESCRIPTION
**Pull request by aidvu**
Add a future proof method for checking if an Atomic site is on the eCommerce plan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,7 @@ add_filter( 'jetpack_tools_to_include', function( $tools ) {
 } );
 ```
 
-To make the bridge work, the site must have the ecommerce plan option set under the `at_options` option:
-
-```
-update_option( 'at_options', array( 'plan_slug' => 'ecommerce' ) );
-```
+To make bridge work, the site must have the eCommerce plan.
```
Comment (aidvu) [inline README.md:43]: Not sure how you test. Another option would be to simply create a small `Atomic_Plan_Manager` class with the needed method, and set a constant and make `current_plan_slug` always return it.

e.g.
```
class Atomic_Plan_Manager {
	const ECOMMERCE_PLAN_SLUG  = 'ecommerce';

	public static function current_plan_slug() {
		return self::ECOMMERCE_PLAN_SLUG;
	}
}

```

**Comment by aidvu** [comment]
👋🏻 @timmyc any chance for a look on this?

**Review by timmyc — APPROVED** [review]
Changes look great to me @aidvu - when do we need to get this updated in wpcomsh?

**Comment by aidvu** [comment]
I'll prepare the version bump stuff, and we can update whenever.

**Review by timmyc — APPROVED** [review]
Version bumps look good, we are going to merge and tag this release.

**Pull request merged**